template/sbt.go

```diff
@@ -13,7 +13,7 @@ sbtscratch:
     {{.ScratchVolumes}}
 {{end}}
 sbt: &sbt
-  image: arch-docker.eng.lancope.local:5000/sbt
+  image: test.local:5000/sbt
```

Comment:
Yeah, this is unfortunate, but there is no official sbt image out there right now, thus the hardcoding of this image and recommending everyone override it: https://github.com/cisco/elsy/blob/master/docs/templates.md#sbt.

We can't remove this until we support something like pluggable templates (maybe via: https://github.com/hashicorp/go-plugin ).